### PR TITLE
Registration completes #36

### PR DESCRIPTION
**Pull request by dralley**

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
```
Comment (rpkilby) [inline accounts/models.py]: Please

File: accounts/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (rpkilby) [inline accounts/tests.py]: Delete

File: accounts/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```
Comment (rpkilby) [inline accounts/views.py]: Me

File: templates/registration/activation_form.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{% extends "labsite.html" %}
+
+{% block title %}Activate{% endblock %}
+{% block content %}
+  <p></p>
```
Comment (rpkilby) [inline templates/registration/activation_form.html]: These <p> tags seem lost

File: templates/registration/invitation_complete.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{% extends "labsite.html" %}
+
+{% block title %}Invitation Complete{% endblock %}
+{% block content %}
+  <p></p>
```
Comment (rpkilby) [inline templates/registration/invitation_complete.html]: Same here

**Comment by dralley** [comment]
@rpkilby Needs review

**Pull request merged**